helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```
@@ -2,22 +2,45 @@ additional:
 {{- if .Values.networkPolicies.additional }}
   {{ toYaml .Values.networkPolicies.additional | nindent 2 }}
 {{- end }}
+
+global:
+  ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
+  apiserver:
+    ips: {{- toYaml .Values.networkPolicies.global.apiserver.ips | nindent 6 }}
+    port: {{ .Values.networkPolicies.global.apiserver.port }}
+  scNodes:
+    ips: {{- toYaml .Values.networkPolicies.global.scNodes.ips | nindent 6 }}
+  wcIngress:
+    ips: {{- toYaml .Values.networkPolicies.global.wcIngress.ips | nindent 6 }}
+  objectStorage:
+    ips: {{- toYaml .Values.networkPolicies.global.objectStorage.ips | nindent 6 }}
+    port: {{ .Values.networkPolicies.global.objectStorage.port }}
+
 harbor:
-  enable: {{ .Values.networkPolicies.harbor.enable }}
-  chartmuseum:
-    egressIPs: {{- toYaml .Values.networkPolicies.harbor.chartmuseum.egressIPs | nindent 6 }}
-  core:
-    egressIPs: {{- toYaml .Values.networkPolicies.harbor.core.egressIPs | nindent 6 }}
+  enabled: {{ .Values.networkPolicies.harbor.enabled }}
```

```suggestion
  enabled: {{ and .Values.harbor.enabled .Values.networkPolicies.harbor.enabled }}
```
Can you add these checks as well? (It will apply to fluentd too since it can also be disabled.)